js/src/ui/Editor/mode-solidity.js

```diff
@@ -22,6 +22,12 @@
 /* eslint-disable */
 var ace = window.ace;
 
+if (process.env.NODE_ENV === 'test') {
```

Comment:
`npm run test` fix - really not sure why it only pops up now and why the rest of the `window.ace` object is defined but not `define(...)`